config/sewing-kit/index.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Package,
+  createComposedProjectPlugin,
+  createProjectTestPlugin,
+  createProjectBuildPlugin,
+} from '@sewing-kit/plugins';
+import {react} from '@sewing-kit/plugin-react';
```

Comment:
Something to keep in mind for the future. It would be nice to have some of these common plugins re-exported  from `@sewing-kit/plugins` so devs can do something like: 

```tsx
import {
   // other stuff
  react, 
  javascript, 
  typescript,
 } from '@sewing-kit/plugins';
```

instead of 

```tsx
import {react} from '@sewing-kit/plugin-react';
import {javascript} from '@sewing-kit/plugin-javascript';
import {typescript} from '@sewing-kit/plugin-typescript';
```

@lemonmade do you foresee any issues with this?

Reply:
I agree the core plugins are annoying to remember what all the package names are

Reply:
This is what composed plugins are for (e.g., https://github.com/lemonmade/quilt/blob/main/packages/sewing-kit-plugins/src/index.ts#L29), so I don't think it's needed.